scripts/reset_uploads.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+PROJECT_DIR="/Users/greg/Library/Application Support/com.tauri.dev/project-x"
```

Comment:
@gjreda was this a temp reset script or should we have one and make it user-agnostic?